src/dotnet/SlnFileExtensions.cs

```diff
@@ -70,6 +70,13 @@ public static void AddProject(this SlnFile slnFile, string fullProjectPath)
                     return;
                 }
 
+                // Identify the intended solution folders
+                IList<string> solutionFolders;
+                if (!string.IsNullOrEmpty(relativeRoot))
```

Comment:
We generally prefer always using curly braces around if-else statements, both as a style and that it's easier to maintain.